package.json

```diff
@@ -13,7 +13,7 @@
   "typings": "lib/analyzer.d.ts",
   "scripts": {
     "clean": "mkdir -p lib/ && rm -rf lib/ && mkdir -p lib/test && cp -r src/test/static lib/test/",
-    "build": "tsc",
+    "build": "tsc || true",
```

Comment:
So the important thing, IMO, is that travis and `prepublish` should only pass if the compiler has no warnings and the tests pass. For development I agree that we should still run tests even if the compiler isn't happy.

Maybe have a `quicktest` npm script that does a permissive compile and then runs tests, and we can use that during development?